docker-compose.yml

```diff
@@ -14,3 +14,4 @@ services:
       io.balena.features.balena-api: '1'
       io.balena.features.dbus: '1'
       io.balena.features.balena-socket: '1'
+      io.balena.features.kernel-modules: '1'
```

Comment:
This does not have any effect currently, but when the Supervisor is able to upgrade itself, it can bind mount /lib/modules and we can run modprobe through the bind mount instead of relying on the `/mnt/root/lib/modules` symlink in `entry.sh`. See TODO note [here](https://github.com/balena-os/balena-supervisor/pull/1966/files#diff-54a2cc8e7ef0905d7ef106e7af17d918725cb359ff1b082db64d2f1e7e82f30cR45-R47)